cypress/fixtures/products.json

```diff
@@ -0,0 +1,10 @@
+{
+  "products": [
```

Comment:
since we are testing the visibility of a menu and not one single product, please add at least one more product here